src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1073,16 +1115,42 @@ private static void CheckUnresolvedBranches(ArrayBuilder<BasicBlock> blocks, Poo
                 return;
             }
 
+            PooledHashSet<BasicBlock> unresolved = null;
             foreach (BasicBlock labeled in labeledBlocks.Values)
             {
                 if (labeled.Ordinal == -1)
                 {
-                    // Neither VB nor C# produce trees with unresolved branches 
-                    // PROTOTYPE(dataflow): It looks like we can get here for blocks from script if block doesn't include all code.
```

Comment:
> Consider keeping part of this comment, explaining that this happens in script code.

There is a PROTOTYPE comment describing this very issue in details in src\Test\Utilities\Portable\Compilation\CompilationExtensions.cs. I think there is no need to doplicate this information now since we don't throw here any more.